src/main/scala/dsptools/Driver.scala

```diff
@@ -8,23 +8,39 @@ import firrtl.{HasFirrtlOptions, ExecutionOptionsManager}
 import numbers.DspRealFactory
 import firrtl_interpreter._
 
+import scala.util.DynamicVariable
+
 
 object Driver {
-  def execute[T <: Module](
+
+  private val optionsManagerVar = new DynamicVariable[Option[VerboseDspTesterOptionsManager]](None)
+  def optionsManager = optionsManagerVar.value.getOrElse(new VerboseDspTesterOptionsManager)
+
+  def execute[T <: Module, M <: TesterOptionsManager](
```

Comment:
Hm but it doesn't hurt to imply that it can be any subclass of TesterOptionsManager? But sure...